packages/docs/PropTables/DropdownPropTable.tsx

```diff
@@ -207,7 +223,9 @@ const dropdownItemGroupProps: Prop[] = [
     required: true,
     description: (
       <>
-        Accepts an array of <Code>DropdownItems</Code> and <Code>DropdownLinkItems</Code>. See example for usage.
+        Accepts an array of <Code>DropdownItems</Code> and <Code>DropdownLinkItems</Code>. See{' '}
+        <NextLink href={{ query: 'implementation=item-groups' }}> example </NextLink>
```

Comment:
I'm ballparking with this suggestion, not sure this is what prettier would want, but let's remove the spaces inside elements.
```suggestion
        <NextLink href={{ query: 'implementation=item-groups' }}>example</NextLink>{' '}
```